packages/components/src/components/input-field/input.tsx

```diff
@@ -111,37 +146,4 @@ const Input = ({
     );
 };
 
-Input.propTypes = {
-    ariaLabel: PropTypes.string,
-    changeValue: PropTypes.func,
-    checked: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-    className: PropTypes.string,
-    classNameInlinePrefix: PropTypes.string,
-    classNameDynamicSuffix: PropTypes.string,
-    current_focus: PropTypes.string,
-    data_testid: PropTypes.string,
-    data_tip: PropTypes.string,
-    data_value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-    display_value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
-    fractional_digits: PropTypes.number,
-    id: PropTypes.string,
-    inline_prefix: PropTypes.string,
-    is_autocomplete_disabled: PropTypes.bool,
-    is_disabled: PropTypes.string,
```

Comment:
as I see, somewhere for <input> and got <inputField> we pass is_disabled as a string, and somewhere as a boolean. 
According to the logic, mybe we should pass it everywhere as a boolean? it would more readable and explicit

Reply:
@yauheni-deriv, I really would rather not change any existing logic for fear of breaking the functionality, or having more changes than there should be in the PR, but I'll look into this and see 😄

Reply:
I believe properties should be named correctly, and used correctly.  if we here find the error, we better should fix, and add qa checklist accordingly, then stay with wrong code

Reply:
I've changed it @yauheni-deriv hopefully, it causes no issues

Reply:
thanks man, @jim-deriv. just please try to mention something in QA checklist